xla/service/gpu/transforms/gemm_rewriter_test.cc

```diff
@@ -5032,6 +5032,58 @@ TEST_P(ParameterizedFp8GemmRewriteTest, UnscaledABUnscaledDMatrixBiasF8) {
       )");
 }
 
+TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABUnscaledDColMajorLhsF8) {
```

Comment:
The test is failing. Can you rebase and check whether you can reproducer the failure?

Reply:
Yes. The failure is reproducible after rebasing. The cause is that the original shape `f8e4m3fn[2,16,32]{1,0,2}` is transformed into `f8e4m3fn[32,2,16]{2,1,0}` before entering `TransposeMatrrix` in gemm_rewriter by pass `hlo_normalization.after_transpose-folding`. This makes the test case not valid. Updated.